### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by liberodark**
A minor improvement

### CODE REVIEW

File: build_wine.sh [hunk 1]
```diff
@@ -315,8 +315,8 @@ export CROSSCFLAGS="${CROSSCFLAGS_X64}"
 export CROSSCXXFLAGS="${CROSSCFLAGS_X64}"
 
 mkdir "${BUILD_DIR}"/build64
-cd "${BUILD_DIR}"/build64
-${BWRAP64} "${BUILD_DIR}"/wine/configure --enable-win64 ${WINE_BUILD_OPTIONS} --prefix "${BUILD_DIR}"/wine-${BUILD_NAME}-amd64
+cd "${BUILD_DIR}"/build64 || exit
+${BWRAP64} "${BUILD_DIR}"/wine/configure --enable-win64 "${WINE_BUILD_OPTIONS}" --prefix "${BUILD_DIR}"/wine-"${BUILD_NAME}"-amd64
```
Comment (Kron4ek) [inline build_wine.sh:319]: WINE_BUILD_OPTIONS should be without quotes.

File: build_wine.sh [hunk 2]
```diff
@@ -328,8 +328,8 @@ export CROSSCFLAGS="${CROSSCFLAGS_X32}"
 export CROSSCXXFLAGS="${CROSSCFLAGS_X32}"
 
 mkdir "${BUILD_DIR}"/build32-tools
-cd "${BUILD_DIR}"/build32-tools
-PKG_CONFIG_LIBDIR=/usr/lib/i386-linux-gnu/pkgconfig:/usr/local/lib/pkgconfig ${BWRAP32} "${BUILD_DIR}"/wine/configure ${WINE_BUILD_OPTIONS} --prefix "${BUILD_DIR}"/wine-${BUILD_NAME}-x86
+cd "${BUILD_DIR}"/build32-tools || exit
+PKG_CONFIG_LIBDIR=/usr/lib/i386-linux-gnu/pkgconfig:/usr/local/lib/pkgconfig ${BWRAP32} "${BUILD_DIR}"/wine/configure "${WINE_BUILD_OPTIONS}" --prefix "${BUILD_DIR}"/wine-"${BUILD_NAME}"-x86
```
Comment (Kron4ek) [inline build_wine.sh:332]: WINE_BUILD_OPTIONS should be without quotes.

File: build_wine.sh [hunk 3]
```diff
@@ -229,35 +229,35 @@ elif [ "$WINE_BRANCH" = "proton" ]; then
...
 		else
-			staging_patcher=("${BUILD_DIR}"/wine-staging-${WINE_VERSION}/staging/patchinstall.py)
+			staging_patcher=("${BUILD_DIR}"/wine-staging-"${WINE_VERSION}"/staging/patchinstall.py)
 		fi
 
-		cd wine
+		cd wine || exit
 		if [ -n "${STAGING_ARGS}" ]; then
-			"${staging_patcher[@]}" ${STAGING_ARGS}
+			"${staging_patcher[@]}" "${STAGING_ARGS}"
```
Comment (Kron4ek) [inline build_wine.sh:260]: STAGING_ARGS should be without quotes.

**Review by Kron4ek — CHANGES_REQUESTED** [review]

**Comment by liberodark** [comment]
Hi,
Sorry I was unfortunately unavailable. 
Thanks for correcting the errors.

**Pull request merged**